x-pack/test/functional/services/ml/navigation.ts

```diff
@@ -88,6 +88,10 @@ export function MachineLearningNavigationProvider({
       await this.navigateToArea('~mlMainTab & ~settings', 'mlPageSettings');
     },
 
+    async navigateToCalendarManagement() {
```

Comment:
Wondering, why this new method is in. It seems it isn't used, also it's doing the same as `navigateToSettings` and doesn't actually navigate to the calendar management.